### ISSUE TITLE
Add the ability to react to or dislike a post/reply

### ISSUE THREAD

**Issue by glen-84** [bluesky-social/social-app#6643]
### Describe the Feature

As it stands, there is no way to react to a message in a non-positive way (other than through replies, which just add noise, and are often not seen).

This means that a post that may otherwise have 100 positive reactions and 50 negative reactions, is simply shown as having 100 positive reactions. There is no "consequence" to posting low quality messages, as the author only receives positive feedback.

Now it's not just a simple task of slapping on a dislike button and calling it a day. If the first reaction to a message is negative, that can sometimes influence others, and you quickly end up with a negative "score" in a way that is not necessarily fair. It can also be abused.

To counteract this, I would suggest that negative reactions are not immediately applied – you can dislike a post/reply, but only after a certain amount of time has passed (or a certain number of reactions have been received), would the dislike count be displayed and applied to the "score".

This can be implemented in many different ways, and I won't go into the algorithmic details unless there is some interest in the general idea.

While something like `score = MAX((likes - dislikes), 0)` is always an option (and an improvement over nothing at all, especially if it's still possible to view the dislike count in a tooltip or similar, or at least via an API), I do think that it should be possible to "bury" particularly bad posts with a negative score after some amount of time has passed.

With regard to the implementation, there are several options:

1. ❤️ / 💔 (likes and dislikes)
2. ⬆️ / ⬇️ (up votes and down votes)
3. One or two of multiple emojis (similar to GitHub).
4. One or two of many emojis (similar to GitLab).

(FWIW, my preference would probably be for `#1` or `#2`. Keeping it simple.)

Assuming `#1` for example purposes, it may show as:

Message posted recently ...

1. ❤️ 5
    - Tooltip: 5 likes.
    - (2 dislikes not yet shown/applied to score)
2. ❤️ 5 💔

Message posted some time ago (period of time or reaction count has been reached) ...

1. ❤️ 50 💔
    - Tooltip: 100 likes, 50 dislikes.
2. ❤️ 100 💔 50

Note that one side-effect of deferred scoring is that the score will often go backwards once the dislikes are applied. However, likes are not currently static anyway – people can remove their likes. This shouldn't be very noticeable on good quality posts.

This dislike data can also feed into the algorithms, for example, reducing the exposure of low quality (highly disliked) posts.

In summary, people should be able to express their opinion, and opinions are not always positive. The way that things currently work is unrealistic – it's not how the real world works. If you shout out something silly IRL, the reactions will not all be positive. Let's make people think more before they post.
